go.work

```diff
@@ -0,0 +1,2 @@
+use .
+use ./v2
```

Comment:
`go.work` is meant to locally modify your workspace. It shouldn't be committed and shouldn't be part of a release. It's generally recommended to add it to `.gitignore`.

Reply:
Thank you!